### PR TITLE
Update TOC for Docker Store

### PR DESCRIPTION
**Pull request by gbarr01**
- Update TOC with entries for new Docker Store docs (BYOL and Publisher FAQ).
- Rearrange TOC to mirror upcoming changes to Docker Store docs.

cc: @KickingTheTV

### CODE REVIEW

File: _data/toc.yaml [hunk 1]
```diff
@@ -2964,13 +2964,21 @@ manuals:
...
-  - path: /docker-store/faq/
-    title: Consumer FAQs
+    title: About Docker Store
+  - sectiontitle: Docker Store FAQs
+    section:
+    - path: /docker-store/customer_faq/
+      title: Customer FAQs
+    - path: /docker-store/publisher_faq/
+      title: Publisher FAQs
+  - sectiontitle: Publisher
```
Comment (mistyhacks) [inline _data/toc.yaml]: I think this title should be "Publish content" or something like that. Or I suppose Publisher Guide maybe. @johndmulhausen for opinion

Comment (gbarr01) [reply _data/toc.yaml]: What do you think @KickingTheTV?

Comment (gbarr01) [reply _data/toc.yaml]: @mistyhacks , what about `For Publishers`? This section covers more than content (and there are plans to add more sub-sections).

**Comment by gbarr01** [comment]
@mistyhacks , thoughts on why this is failing?

**Comment by mistyhacks** [comment]
It is saying that the tag doesn't exist but it does --- https://cloud.docker.com/app/docs/repository/docker/docs/docker.github.io/tags/docs-builder maybe a race condition? You can do an amend and force-push in the PR branch and it will make Jenkins run again. The Jenkins tests are not show-stoppers for merging, though.

**Pull request merged**